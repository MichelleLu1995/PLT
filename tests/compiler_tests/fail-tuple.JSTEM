def void main() {
	float(%8%) t;
	int j;

	t = (% 1.0, 2.0, 3.0, 4.0, 5.0, 6.0, 7.0, 8.0 %);

	for (j=0; j<8; j=j+1){
		printf(t(%j%));
	}
}
