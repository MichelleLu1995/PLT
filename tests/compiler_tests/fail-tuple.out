Fatal error: exception Failure("illegal tuple type")